lib/dovico/api_client.rb

```diff
@@ -28,6 +30,21 @@ def delete(path, params: {}, body: nil)
         perform!(:delete, path, params: params, body: body)
       end
 
+      def get_paginated_list(path, object_key, **params)
+        results = get(path, **params)
+        objects = results[object_key]
+
+        loop do
+          next_page_uri = results["NextPageURI"]
+          break if next_page_uri.nil? || next_page_uri == "N/A"
```

Comment:
:thinking: Should we check if `next_page_uri` doesn't look like an URL ?

Reply:
Well, the possible values are : 
- an URL
- `N/A`

Maybe some API calls does not include the field.

So I find more easy to just test it's exist and it's not `N/A`. I think we can trust the Dovico API here.